packages/backend/src/index.ts

```diff
@@ -28,28 +28,38 @@ import {
   notFoundHandler,
   requestLoggingHandler,
 } from '@backstage/backend-common';
-import { router as inventoryRouter } from '@backstage/plugin-inventory-backend';
+import { createRouter as inventoryRouter } from '@backstage/plugin-inventory-backend';
 import compression from 'compression';
 import cors from 'cors';
 import express from 'express';
 import helmet from 'helmet';
 import { testRouter } from './test';
 
 const DEFAULT_PORT = 7000;
-
 const PORT = parseInt(process.env.PORT ?? '', 10) || DEFAULT_PORT;
-const app = express();
+const logger = getRootLogger().child({ type: 'plugin' });
+
+async function main() {
+  const app = express();
+
+  app.use(helmet());
+  app.use(cors());
+  app.use(compression());
+  app.use(express.json());
+  app.use(requestLoggingHandler());
+  app.use('/test', testRouter);
+  app.use(
+    '/inventory',
+    await inventoryRouter({
+      logger,
+    }),
+  );
+  app.use(notFoundHandler());
+  app.use(errorHandler());
 
-app.use(helmet());
-app.use(cors());
-app.use(compression());
-app.use(express.json());
-app.use(requestLoggingHandler());
-app.use('/test', testRouter);
-app.use('/inventory', inventoryRouter);
-app.use(errorHandler());
-app.use(notFoundHandler());
+  app.listen(PORT, () => {
+    getRootLogger().info(`Listening on port ${PORT}`);
```

Comment:
why not `logger.info` here? (just trying to understand the logging strat)

Reply:
Yeah it's a little confusing to have both in the same file - the idea here was that it doesn't make sense to log with a tag `type: 'plugin'`.